modules/local/dada2_filtntrim.nf

```diff
@@ -1,6 +1,8 @@
 process DADA2_FILTNTRIM {
     tag "$meta.id"
     label 'process_medium'
+    errorStrategy 'retry'
+    maxRetries 3
```

Comment:
This is [already set](https://github.com/nf-core/ampliseq/blob/8f139cecd95a23920617f2c2b154733494ed4562/conf/base.config#L19) globally at `process` level, so no need to set again.
```suggestion
```